### PR TITLE
Incorporate ontology review changes to PROV-S

### PR DESCRIPTION
**Pull request by glguy**

### CODE REVIEW

File: RACK-Ontology/ontology/PROV-S.sadl [hunk 1]
```diff
@@ -10,41 +10,57 @@
...
+	invalidatedAtTime describes ENTITY with at most 1 value.
 
 COLLECTION (note "A collection is an entity that provides a structure to some constituents, which are themselves entities. These constituents are said to be member of the collections.")
 	is a type of ENTITY.
 	content (note "A collection is an entity that provides a structure to some constituents, which are themselves entities. These constituents are said to be member of the collections.") describes COLLECTION with values of type ENTITY.
 
 AGENT (note "An agent is something that bears some form of responsibility for an activity taking place, for the existence of an entity, or for another agent's activity.")
 	is a type of THING.
+	agentName describes AGENT with values of type string.
+	agentName describes AGENT with at most 1 value.
```
Comment (russell-d-e) [inline RACK-Ontology/ontology/PROV-S.sadl:54]: for agentName should we consider having multiple agentNames for situations like if they wanted to capture first/last name or aliases for things like the same person had a name change (i.e. maiden names)

**Review by russell-d-e — COMMENTED** [review]
had comment on the cardinality for agentName

**Comment by AbhaMoitra** [comment]
@davearcher - you are the reviewer and since we agreed on this, can you review / approve this request AFTER the conflict is resolved.

File: RACK-Ontology/ontology/PROV-S.sadl (hunk 1 above)
Comment (davearcher) [reply RACK-Ontology/ontology/PROV-S.sadl:54]: Proposing to drop agentName altogether in favor of using Title in the base class.

**Comment by davearcher** [comment]
I think this one is good to go, after agentName is removed by @glguy

**Pull request closed without merging**